concepts/arrays-and-lists/about.md

```diff
@@ -56,7 +56,7 @@ scalar @names; # 3
 scalar ('Alice', 'Bob', 'Charlie'); # 'Charlie' (and some warnings).
 ```
 
-Read more in the folowing FAQ: [What is the difference between a list and an array?](perlfaq4listarray)
+Read more in the following article: [What is the difference between a list and an array?](https://perldoc.pl/perlfaq4#What-is-the-difference-between-a-list-and-an-array?)
```

Comment:
```suggestion
Read more in the following article: [What is the difference between a list and an array?][perlfaq4listarray]
```

Shouldn't it be like this?

Reply:
Will that make the shortcode work? (perlfaq4listarray) 

I'm not really sure how the shortcodes work, so I was just putting the perlfaq4 there

Reply:
The square brackets and then square brackets would work to give a display that is different than the link reference.

The URL did not change, and so I think that the use of `()` was the problem ,rather than the URI that it is meant to link to.

Reply:
You can even give it a try in the github comment area and hit preview to see what you see.

Reply:
I'm sure I'm just clicking the wrong place, but it says 'Nothing to Preview' everywhere I've checked.

I'll just update the PR with the requested changes; can always update some more if that doesn't fix it (which I would imagine it will based on your familiarity with this)